models/ParentRegistration.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Thoughts on naming file `Registration.ts` and having `VolunteerRegistrationInput` in here as well?

Reply:
Named it to `Enroll.ts` to be consistent with the resource naming